### PR TITLE
Prevents effects from feeding the singulo

### LINKED ISSUE #23496: Sparks can feed the singulo
>Dumbshit engineers don't anchor tesla machinery down
>Sparks feed singulo
>Wow, 15 minute extended round.

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #23496 

:cl: Cyberboss
fix: Abstract entities no longer feed the singularity
/:cl:

### CODE REVIEW

File: code/game/objects/effects/effects.dm [hunk 1]
```diff
@@ -42,3 +42,7 @@
 
 /obj/effect/experience_pressure_difference()
 	return
+
+/obj/effect/singularity_act()
+	..()
+	return 0
```
Comment (coiax) [inline code/game/objects/effects/effects.dm]: Make this . = 0

**Comment by Kromgar** [comment]
No more sparks making stage 5 singulos?

**Comment by Hatfish** [comment]
👎 Engineers should know to secure all heavy machinery in the first place.

File: code/game/objects/effects/effects.dm (hunk 1 above)
Comment (MrPerson) [inline code/game/objects/effects/effects.dm]: Why? This is literally the same thing

**Comment by F-OS** [comment]
:-1: x :100: Feature.

**Comment by Cyberboss** [comment]
It's a fucking bug you plebians

**Comment by Haevacht** [comment]
Make it give fuckall food but let sparks feed it. Punish shitty engineers.

**Comment by Haevacht** [comment]
Make it give fuckall food but let sparks feed it. Punish shitty engineers.

File: code/game/objects/effects/effects.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/objects/effects/effects.dm]: I doubt this should even call the parent

**Comment by optimumtact** [comment]
This isn't a feature

**Comment by MrStonedOne** [comment]
\>spiders don't feed it

\>posters don't feed it

\>sparks don't feed it

\>mines don't feed it

**Comment by optimumtact** [comment]
hmm

**Comment by MrStonedOne** [comment]
If they must make this retarded PR happen, add it to effects/particle_effects

**Comment by optimumtact** [comment]
really jogs my nog

**Comment by PKPenguin321** [comment]
Why are all of those effects in the first place aaaa

**Comment by ChangelingRain** [comment]
\>spiders are structures

\>posters are structures

\>sparks are the reason this pr exists

\>mines are still effects for some reason and also happen to be admin-only

**Comment by optimumtact** [comment]
you can't do this for the reasons mentioned, perhaps a typecache or something?

**Comment by ChangelingRain** [comment]
Uh, but only one of those reasons is even half valid????
And that reason is mines, which are effects for some godawful reason.
And, oh, my, mines are admin-only and thus shouldn't matter.

**Comment by ChangelingRain** [comment]
My tl;dr is that MSO's points are basically either all wrong or about things that are admin-only and should not contribute enough to the singularity for it to matter.

**Pull request merged**